testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -2286,6 +2286,158 @@ public void deleteColumnPartitionKey() throws IOException {
     assertThat(response.getDescription()).isNotEmpty();
   }
 
+  @Test
+  public void createUdt() throws IOException {
```

Comment:
Related to my comments above, it might be good to add tests for bad cql types and missing field definitions.

Reply:
Have included tests for bad types and absent type definitions.